app/bundles/EmailBundle/Config/config.php

```diff
@@ -612,6 +615,12 @@
                 'arguments' => [],
             ],
         ],
+        'helpers' => [
+            'mautic.helper.form.asubmit' => [
+                'class'     => \Mautic\EmailBundle\Helper\FormSubmitHelper::class,
+                'arguments' => ['mautic.helper.core_parameters'],
+            ],
+        ],
```

Comment:
Yes, you are right. It is a static method so DI won't help us here. It will need some more refactoring to get rid of MauticFactory in this helper. Please remove this and the constructor in that helper.

Reply:
Ok, reverted and ready to re-test